src/main/java/spoon/support/visitor/java/reflect/RtMethod.java

```diff
@@ -137,6 +137,11 @@ public int hashCode() {
 		return getDeclaringClass().getName().hashCode() ^ getName().hashCode();
 	}
 
+	/**
+	 * Creates a new {@link RtMethod}
+	 * @param method is passed as Method
+	 * @return {@link RtMethod}
+	 */
```

Comment:
This Javadoc does not really say anything, it mostly just duplicates the method header.

What this method does is to wrap a `Method` in an `RtMethod`. Note that it's entirely fine to have a Javadoc _without_ a general description. That is to say, like so:

```suggestion
	/**
	 * @param method Say something about the parameter
	 * @return Say something about the return value
	 */
```

Could you try to improve this Javadoc to say something meaningful?